Tests/APITesters/SwiftAPITester/SwiftAPITester/AttributionAPI.swift

```diff
@@ -43,9 +43,6 @@ func checkAttributionAPI() {
     attribution.setOnesignalID("")
     attribution.setOnesignalID(nil)
 
-    attribution.setOnesignalUserID("")
```

Comment:
We need to keep these and the tests because they're still part of the API (so we want to verify the API and have tests).
Instead you can move the API tests to a method that'd also marked with `@available(*, deprecated)`, and mark the tests with the same annotation.